pylint/utils.py

```diff
@@ -1157,6 +1157,12 @@ def _ini_format(stream, options, encoding):
             print('#%s=' % optname, file=stream)
         else:
             value = _encode(value, encoding).strip()
+            if section == 'MESSAGES CONTROL' and ',' in str(value):
```

Comment:
I think this is too specialized. It makes this code have intimate knowledge about the sections we support. Not sure what other solution could we use, but the current approach is not the best one.